client/src/components/shared/Info/InfoButton.js

```diff
@@ -8,6 +8,7 @@ import type {Node} from 'react'
 
 import {ShowNumberCurrency} from '../../../services/utilities'
 import type {Contract, NoticeNew, Eufund} from '../../../state'
+import type {stateUpdater} from '../../../types/commonTypes'
```

Comment:
Me, a nitpicker: why does it start with lowercase when it's a type?

Reply:
Because there is already a `onDataFunction` type next to it. But I gladly rename them both to uppercase.